index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>protfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <p>My Logo</p>
+            <img src="image/Enabled.png">
+        </nav>
+    </header>
+    <main>
+        <section class="intro-section">
+            <h1>Hey There. <br> I’m Shavaiz. </h1>
+            <h3>I’m a Software Developer</h3>
+            <p class="primary-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <div class="social-link">
+                <div class="icons"><img src="image/cat.png" alt=""></div>
+                <div class="icons"><img src="image/linkdin.png" alt=""></div>
+                <div class="icons"><img src="image/victory.png" alt=""></div>
+                <div class="icons"><img src="image/Frame.png" alt=""></div>
+                <div class="icons"><img src="image/icon m.png" alt=""></div>
+            </div>
+        </section>
+    </main>
```

Comment:
@emshavaiz Good Job so far 🆒 I would suggest to adjust bottom padding/margin to make your design similar to Figma 
![shavaiz](https://github.com/emshavaiz/portfolio-pro/assets/122353030/e6a7758c-08c8-42e1-b261-abc140b67674)